client/initializers/underscore.js

```diff
@@ -15,3 +15,69 @@ _.isPresent = function(value) {
 _.isBlank = function(value) {
   return !_.isPresent(value);
 };
+
+// FIXME: Replace this comparator function by server side implementation
+// Based on topic: http://stackoverflow.com/questions/6832596/how-to-compare-software-version-number-using-js-only-number
+_.compareVersion = function(v1, v2) {
+  var versionRegex = /(\d+)?(.*)?/;
```

Comment:
This is not a very clean way to match a semantic versioning component. It would match an empty string, or a string with both a number and text like "42foo", neither of which are valid. I would first check whether each part is a number with a simple `\d+` regexp, and then run the appropriate comparison depending on whether it's a number or not.

There are a lot of other ways in which this function would not sort semantic versions correctly, but we can leave that for another time. I have added a ticket to our board for the server-side implementation.